inc/Engine/Common/Ajax/AjaxHandler.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace WP_Rocket\Engine\Common\Ajax;
+
+class AjaxHandler {
+
+	/**
+	 * Validate the referer before going futher.
+	 *
+	 * @param array $args Arguments to configure the validation.
+	 *
+	 * @return bool
+	 */
+	public function validate_referer( array $args ) {
```

Comment:
I don't like using array much in method parameters, it's hiding what is expected and creates the need to validate the array key/values passed.

For an AJAX request we know what we are going to validate is the nonce/referer and the capability, so we can pass those as parameter directly.